js/kucoin.js

```diff
@@ -2062,50 +2062,27 @@ module.exports = class kucoin extends Exchange {
             const response = await this.futuresPrivatePostTransferOut (this.extend (request, params));
             //
             //     {
-            //         code: '200000',
-            //         data: {
-            //             applyId: '605a87217dff1500063d485d',
-            //             bizNo: 'bcd6e5e1291f4905af84dc',
-            //             payAccountType: 'CONTRACT',
-            //             payTag: 'DEFAULT',
-            //             remark: '',
-            //             recAccountType: 'MAIN',
-            //             recTag: 'DEFAULT',
-            //             recRemark: '',
-            //             recSystem: 'KUCOIN',
-            //             status: 'PROCESSING',
-            //             currency: 'XBT',
-            //             amount: '0.00001',
-            //             fee: '0',
-            //             sn: '573688685663948',
-            //             reason: '',
-            //             createdAt: 1616545569000,
-            //             updatedAt: 1616545569000
-            //         }
+            //         'applyId': '605a87217dff1500063d485d',
```

Comment:
This comment doesn't correspond to the parsing lines below, since a sub-dictionary named `data` is expected. We have to either fix the comment or the parsing code. In this case, I think, we must fix the comment.

Reply:
yeah, it was my mistake, i've copied the single  structure comment into `parseTransfer` and had copied & overwriten it back. 👍